docs/operator-guide/cd/manual-approval.md

```diff
@@ -0,0 +1,109 @@
+
+## Manual Approval Structure
+
+A deploy pipeline with a manual approval step is a typical Tekton pipeline but with an [ApprovalTask](https://github.com/epam/edp-tekton/blob/48f0d18ecfec049618999dabd9c8cbc892ae3508/charts/pipelines-library/templates/pipelines/cd/deploy-with-approve.yaml#L61) custom resource included. The custom deploy pipeline usage flow involves the following steps:
+
+```mermaid
+graph LR;
+    A(Create TriggerTemplate resource) --> B(Create Pipeline resource) --> C(Apply changes to environment) --> D(Run deploy pipeline) --> E(Approve/reject deployment)
+```
+
+## Create Pipeline With Approval Task
+
+To create a deploy pipeline with a manual approval task, follow the steps below:
+
+1. Create and apply the **TriggerTemplate** custom resource. Refer to the example [TriggerTemplate](https://github.com/epam/edp-tekton/blob/48f0d18ecfec049618999dabd9c8cbc892ae3508/charts/pipelines-library/templates/triggers/cd/deploy-with-approve.yaml). Make sure to specify the correct pipeline type in the labels and provide the pipeline reference in the specifications:
+
+  ```bash
+  ...
+  labels:
+      app.edp.epam.com/pipelinetype: deploy
+  ...
+  ```
+
+  ```bash
+  ...
+  spec:
+    taskRunTemplate:
+      serviceAccountName: tekton
+    pipelineRef:
+      name: deploy-with-promote-approval
+  ...
+  ```
+
+2. Create and apply the **Pipeline** custom resource. Refer to the example [Pipeline](https://github.com/epam/edp-tekton/blob/48f0d18ecfec049618999dabd9c8cbc892ae3508/charts/pipelines-library/templates/pipelines/cd/deploy-with-approve.yaml). Ensure to set the concrete task to trigger:
+
+  Additionally, set the concrete approval task to run:
+
+  ```bash
+  - name: approve
+    params:
+      - name: description
+        value: Promote versions to the next Environment?
+    runAfter:
+      - deploy-app
+    taskRef:
+      apiVersion: edp.epam.com/v1alpha1s
+      kind: ApprovalTask
+      name: approve
+  ```
+
+    :::note
+        KubeRocketCI also offers a pre-defined deploy pipeline called **deploy-with-approve**. It contains an approval task and can be used for most application deployment cases.
+    :::
+
+## Enable Deploy Pipeline
+
+Once all the custom resources are correctly configured and applied, you can select the newly created pipeline in the KubeRocketCI portal.
+
+### New Environments
+
+To apply the newly created deploy pipeline to your environments, follow the steps below:
+
+1. In the deployment flow details page, click the **Create environment** button:
+
+  ![Create environment button](../../assets/operator-guide/create-environment-button-deploy.png "Create environment button")
+
+2. In the **Create environment** window, select the custom deploy pipeline:
+
+  ![Select pipeline with approval](../../assets/operator-guide/custom-approve-pipeline1.png "Select pipeline with approval")
+
+### Existing Environments
```

Comment:
Also I have parameter to approve with comment, do we need to describe this case?